Source/drlg_l4.cpp

```diff
@@ -1283,11 +1283,49 @@ void GeneralFix()
 	}
 }
 
+bool PlaceMiniSets(lvl_entry entry)

```

Comment:
each version of this function can be called something like `PlaceStairs`, bit more descriptive since they all deal with a specific set of minisets.

Reply:
Damn, thats exactly what I called it when i started on drlg_l4, but decided to rename it back to `PlaceMiniSets` :D

Reply:
Fixed and rebased